keyboards/handwired/videowriter/config.h

```diff
@@ -20,12 +20,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID 0xFEED
-#define PRODUCT_ID 0x0000
+#define VENDOR_ID 0x515A // "QZ"
+#define PRODUCT_ID 0x5657 // "VW"
 #define DEVICE_VER 0x0001
 #define MANUFACTURER Magnavox
-#define PRODUCT Videowriter
-#define DESCRIPTION Keyboard
+#define PRODUCT Magnavox Videowriter
```

Comment:
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs
> don’t repeat MANUFACTURER in the PRODUCT value

```suggestion
#define PRODUCT Videowriter
```